examples/with-firebase-hosting/package.json

```diff
@@ -13,15 +13,15 @@
     "logs": "firebase functions:log"
   },
   "dependencies": {
-    "firebase-admin": "^8.9.0",
-    "firebase-functions": "^3.3.0",
-    "next": "^9.3.4",
+    "firebase-admin": "^9.4.2",
+    "firebase-functions": "^3.13.1",
+    "next": "^9.5.5",
```

Comment:
Any reason not to update to the latest version?

Reply:
This can probably just be set to `"next": "latest"` I cannot remember why we stopped doing that in this example, though I imagine the root issue is fixed or no longer relevant

Reply:
I'd say let's update all the deps in this example, make sure it works, and then it's good to merge!

Reply:
You are right! I've updated to `next@latest` and use react@17.0.1

Reply:
Thank you!